src/txn/txn_recover.c

```diff
@@ -988,7 +988,7 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
          * Forcibly log a checkpoint so the next open is fast and keep the metadata up to date with
```

Comment:
This comment is now out of date since it is no longer "Forcibly". Maybe something like:
Take a system-wide checkpoint of all recovered tables so that the next open is fast...

Reply:
Fixed in [81173dc](https://github.com/wiredtiger/wiredtiger/pull/8160/commits/81173dc5f8d9b39d43926c03a0a86cc5bcb132d5). Also merged with develop since the branch had grown quite stale.